### ISSUE TITLE
questions of RFM prediction results

### ISSUE THREAD

**Issue by FoxLouis1** [panpiort8/ReactionFeasibilityModel#1]
I'd like to express my gratitude for your outstanding work 'RetroGFN'. It has been incredibly helpful in my research.This is the prediction results for the example in the notebook using your checkpoint. We notice that the output contains negative values, which contradicts the convention that RFM(x) ∈ [0, 1] in the paper of RetroGFN.Besides, I didn't observe any normalization to [0, 1] in the MLP layer of model. Could you please explain how you constructed the labels and trained the model, and provide the corresponding code?
![b17fb9e7aa69d2498f7c6b55551cb01](https://github.com/user-attachments/assets/d75db626-c85f-4e83-bbfd-8abcea5d45ea)

**Comment by panpiort8** [comment]
Hello! Thanks for your interest in our work! The model outputs logits $\in \mathbb{R}$ instead of probabilities $\in [0, 1]$, because it was trained with [BCEWithLogitsLoss](https://pytorch.org/docs/stable/generated/torch.nn.BCEWithLogitsLoss.html). To obtain the probabilities, one needs to call `torch.sigmoid` explicitly. In RetroGFN code it's done [here](https://github.com/gmum/RetroGFN/blob/b12fc894ac44c1d5f778f8d7b72741152c75f530/gflownet/gfns/retro/proxies/retro_rfm_proxy.py#L104). 

I've created a PR #2 that adds the training scripts, configs and data preparation pipeline as suggested. I also added a brief description of the dataset preparation in [update README](https://github.com/panpiort8/ReactionFeasibilityModel/blob/add_training_details/README.md#data-preparation). 

The training was done as binary classification with label $= 1$ meaning a positive reaction coming from the original USPTO-50k and label $= 0$ being a negative reaction constructed with our data preparation pipeline.

If you need some more clarifications, you're welcome to ask!

**Comment by FoxLouis1** [comment]
Thank you very much!
